image/image_test.go

```diff
@@ -54,6 +55,28 @@ func TestMarshalKeyOrder(t *testing.T) {
 	}
 }
 
+func TestImageWithAllMethods(t *testing.T) {
```

Comment:
nit: maybe just `TestImage()`

Reply:
Yes, I think just `TestImage()` would be better